### PR TITLE
Feat/342 adgk china

### PR DESCRIPTION
**Pull request by Alex-302**
Replaced obsolete Prebake filter.
Not sure about trustLevel.

### CODE REVIEW

**Comment by krystian3w** [comment]
Maybe need _high_  to work `mp4`/`empty` moddifier.

```
.atm.cp31.ott.cibntv.net$media,MP4
||valipl.cp31.ott.cibntv.net^$domain=v.youku.com,mp4,media
||cdn-adn.rayjump.com/cdn-adn/*.mp4$app=com.naga.kun,mp4
||vali.cp31.ott.cibntv.net/youku/*/*.mp4*$app=com.huawei.hwvplayer.youku,mp4
||*/youku/*/*.mp4*vali.cp31.ott.cibntv.net*$app=com.huawei.hwvplayer.youku,mp4
||vali.cp31.ott.cibntv.net/youku^$domain=v.youku.com,mp4
```
```
||cpoml.iask.sina.com.cn^$empty
||sl.sogoucdn.com/semob$empty,popup
/?encrypt=$empty,app=com.tencent.mtt.intl
```

**Comment by slavaleleka** [comment]
@krystian3w 
modifiers `mp4` and `empty` are not related to trust levels

**Comment by krystian3w** [comment]
So can work on `low` - no use `+js()` or `redirect=` or  `{ style: css }`.

**Comment by krystian3w** [comment]
I found invalid CSS (`##` instead of `#$#`) - if this no problem for `filterCompler` so need for 1 filter `high` instead of `low`.

`chinanews.com##.enter_top {    width: 100%;    height: 154px;    background: #fff;}`

---

this is strange:

```
(function() {
var hm = document.createElement("script");
var s = document.getElementsByTagName("script")[0];
s.parentNode.insertBefore(hm, s);
})();
```

little JS almost in the middle of the file.

**Comment by ameshkov** [comment]
> little JS almost in the middle of the file.

Yeah, this is quite weird indeed

**Comment by Alex-302** [comment]
Maybe wrong copy-pasta. Looks like a part of Baidu tracker.

**Comment by banbendalao** [comment]
The warehouse has been urgently migrated https://github.com/banbendalao/ADgk

**Comment by Alex-302** [comment]
@banbendalao please take a look https://github.com/banbendalao/ADgk/issues/1

File: filters/ThirdParty/filter_209_ADgkMobileChina/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "filterId": 209,
+  "name": "ADgk Mobile China list",
+  "description": "Filter for mobile Chinese sites.",
+  "timeAdded": 1590425220741,
+  "homepage": "http://adgk.net/",
```
Comment (zloyden) [inline filters/ThirdParty/filter_209_ADgkMobileChina/metadata.json:6]: Now the home page `https://github.com/banbendalao/ADgk`.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by zloyden** [comment]
@Alex-302 please resolve conflicts and review it again. @banbendalao removed problematic rules.

**Pull request closed without merging**